commit 54a04e7fc16c4ca6384b9f40ffea40a8f5c517d2
Author: Michael Still <mikal@stillhq.com>
Date:   Thu Jun 4 05:21:27 2026 +1000

    Master is now Hibiscus OpenStack release.
    
    Previously once a stable branch was cut, the version lock would
    be reverted on master. This pattern can be seen in commits
    87678d014, and c30e4e524. However this time the decision has been
    made to lock in the code name for the next release as well, so this
    change is strictly not a revert.
    
    I specifically found this because Nova master commit 770580bb bumped
    oslo.privsep in requirements.txt from >=2.6.2 to >=3.11.0. With
    OPENSTACK_RELEASE = '2026.1', openstack_branch resolves to
    stable-2026.1 and openstack-base pulls
    requirements-stable-2026.1.tar.gz, which still pins
    oslo.privsep===3.10.1. The resulting ResolutionImpossible causes
    image builds to fail.
    
    Change-Id: I7d156af2fa9677cd4921f518ac1c6ecf7acde9ce
    Signed-off-by: Michael Still <mikal@stillhq.com>

diff --git a/kolla/common/config.py b/kolla/common/config.py
index 8a80e1f88..897520807 100644
--- a/kolla/common/config.py
+++ b/kolla/common/config.py
@@ -38,8 +38,8 @@ DISTRO_PRETTY_NAME = {
     'ubuntu': 'Ubuntu 24.04.* LTS',
 }
 
-OPENSTACK_RELEASE = '2026.1'
-OPENSTACK_RELEASE_CODENAME = 'Gazpacho'
+OPENSTACK_RELEASE = 'master'
+OPENSTACK_RELEASE_CODENAME = 'Hibiscus'
 
 # TODO(mandre) check for file integrity instead of downloading from an HTTPS
 # source
